.github/workflows/build_and_test_workflow.yml

```diff
@@ -116,12 +171,64 @@ jobs:
           JOB: ci${{ matrix.group }}
           CACHE_DIR: ciGroup${{ matrix.group }}
 
-  build-min-artifact-tests:
+  functional-tests-windows:
+    runs-on: windows-latest
+    name: Run functional tests on Windows
+    strategy:
+      matrix:
+        group: [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 ]
+    steps:
+      - run: echo Running functional tests for ciGroup${{ matrix.group }}
+
+      - name: Configure git's autocrlf
+        run: |
+          git config --global core.autocrlf false
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup JDK
+        uses: actions/setup-java@v3
```

Comment:
how come?

Reply:
I am using the bare Windows image which doesn't have JDK to run OS for the tests.